src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
```

Comment:
```suggestion
    name,
    wheels,
    version,
```
Тут назви ключів і змінних однакові. можна використовувати скорочений запис